assembly-utils.gemspec

```diff
@@ -18,15 +18,9 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_dependency 'nokogiri'
-  s.add_dependency 'csv-mapper'
-  s.add_dependency 'fastercsv'
   s.add_dependency 'druid-tools', '>= 0.2.6'
 
-  s.add_dependency 'dor-services', '~> 5.3'
-  s.add_dependency 'dor-workflow-service', '>=1.3.1'
-
-  s.add_dependency 'activesupport'
-  s.add_dependency 'activeresource'
+  s.add_dependency 'dor-services', '~> 5.5'
```

Comment:
d'oh:  i merged, but i'd had a question:  doesn't this still allow the usage of dor-services 5.3?  should a `'>= 5.5'` constraint be added?  i was under the impression that pinning to `'~> N.x'` would allow anything of major version `N` to be used (i've always found that behavior puzzling at best, so i'd be happy to be wrong about the least significant part of the version number being ignored, and i'm not an expert on Gemfile/.gemspec/bundler).